docs/index.md

```diff
@@ -96,6 +96,17 @@ git checkout v7.0.13
 scramv1 b clean; scramv1 b # always make a clean build
 ```
 
+## Oustide of CMSSW (recommended for non-CMS users)
+
+Pre-compiled versions of the tool are available as containers from the [CMS cloud pages](https://gitlab.cern.ch/cms-cloud/combine-standalone). These containers can be downloaded and run using [Docker](https://cms-opendata-guide.web.cern.ch/tools/docker/. If you have docker running you can pull and run the *latest* version using, 
```

Comment:
Do we also want to include instructions for running older versions (i.e. not "latest") or do we prefer to not mention it to keep people hopefully using the most up to date version? It's probably not so relevant yet in either case.

Reply:
For now let's keep it as it is. I suppose we will want to have stamped versions also for the images in the future but we can always update the docs once we know how that will work.